packages/core/src/components/skeleton/_skeleton.scss

```diff
@@ -0,0 +1,95 @@
+// Copyright 2016 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+/*
+Skeletons
+
+Skeletons allow you to show a loading state that mimics the shape of your yet-to-load content.
+
+Weight: 1
+
+Styleguide components.progress.skeleton
+*/
+
+/*
+CSS API
+
+Apply the class `.pt-skeleton` to elements that you would like to cover up with a loading animation. The skeleton inherits the
+dimensions of whatever element the class is applied to. This means that when using skeletons to show loading text, you should use
+some sort of placeholder text that is approximately the length of your expected text.
+
+Markup:
+<div class="pt-card">
+  <h5><a class="{{.modifier}}" href="#">My Card Title</a></h5>
+  <p class="{{.modifier}}">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Quisque eget tortor felis. Fusce dapibus metus in dapibus mollis.
+    Quisque eget ex diam. Vivamus mattis nisi ut nisl rhoncus convallis. Curabitur id tristique mi, quis elementum dolor. In
+    suscipit ut sapien ac tempor. Aenean quis fringilla felis.
+  </p>
+  <button type="button" class="pt-button {{.modifier}}">Submit</button>
+</div>
+
+.pt-skeleton - Covers up your element with a skeleton animation.
+
+Styleguide components.progress.skeleton.css
+*/
+
+@import "../../common/variables";
+
+$skeleton-animation-duration: $pt-transition-duration * 20;
+$skeleton-color-start: rgba($gray4, 0.2);
+$skeleton-color-end: rgba($gray1, 0.2);
```

Comment:
please add `!default` to the end of each of these variables. see #399 for inspiration